python_modules/dagster/dagster/core/instance/__init__.py

```diff
@@ -341,6 +341,16 @@ def __init__(
                 "worker will be marked as failed, but will not be resumed.",
             )
 
+        if self.run_retries_enabled:
+            check.invariant(
+                self.run_storage.supports_kvs(),
+                "Run retries require a run storage that supports KVS",
```

Comment:
this would be very cryptic to me as a user if I saw it - KVS is not a standard acronym and can it include advice on what they should do to fix it?